doctr/utils/data.py

```diff
@@ -70,7 +70,10 @@ def download_from_url(
         file_name = url.rpartition('/')[-1]
 
     if not isinstance(cache_dir, str):
-        cache_dir = os.path.join(os.path.expanduser('~'), '.cache', 'doctr')
+        if os.environ['DOCTR_CACHE_DIR']:
+            cache_dir = os.environ['DOCTR_CACHE_DIR']
+        else:
+            cache_dir = os.path.join(os.path.expanduser('~'), '.cache', 'doctr')
```

Comment:
I'd suggest a simpler version with the get method:
```python
cache_dir = os.environ.get("DOCTR_CACHE_DIR", os.path.join(os.path.expanduser("~"), ".cache", "doctr"))
```

Reply:
LGTM

Reply:
Even better ! :+1:

Reply:
Wow! looks good! I didn't know it accepts default value as second argument
Also maybe it's worth to check if directory is writable and if it's not hint user that they can set `DOCTR_CACHE_DIR` to writable directory. That's what `matplotlib` did and it was really helpful